tests/test_plugins_os_windows_defender.py

```diff
@@ -23,3 +30,91 @@ def test_defender_evtx_logs(target_win, fs_win, tmpdir_name):
+    assert mimikatz_record.last_write_time.date() == detection_date
+    assert mimikatz_record.last_accessed_time.date() == detection_date
+
+
+def test_defender_quarantine_recovery(target_win, fs_win, tmpdir_name):
+    # Map the quarantine folder from our test data
+    quarantine_dir = absolute_path("data/defender-quarantine")
+    fs_win.map_dir("programdata/microsoft/windows defender/quarantine", quarantine_dir)
+
+    # Create a directory to recover to
+    recovery_dst = Path(tmpdir_name).joinpath("recovery")
+    os.mkdir(recovery_dst)
+
+    # Recover
+    target_win.add_plugin(defender.MicrosoftDefenderPlugin)
+    target_win.defender.recover(output_dir=recovery_dst)
+
+    # Set up variables to indicate what we expect to find
+    payload_filename = "A6C8322B8A19AEED96EFBD045206966DA4C9619D"
+    security_descriptor_filename = "A6C8322B8A19AEED96EFBD045206966DA4C9619D.security_descriptor"
+    zone_identifier_filename = "A6C8322B8A19AEED96EFBD045206966DA4C9619D.ZoneIdentifierDATA"
+    expected_zone_identifier = {
+        "ZoneTransfer": {"zoneid": "3", "referrerurl": "C:\\Users\\user\\Downloads\\mimikatz_trunk.zip"}
+    }
+    expected_owner = "S-1-5-21-2614236324-1336345114-3023566343-1000"
+    expected_group = "S-1-5-21-2614236324-1336345114-3023566343-513"
+
+    expected_files = [payload_filename, security_descriptor_filename, zone_identifier_filename]
+    expected_files.sort()
+
+    directory_content = os.listdir(recovery_dst)
+    directory_content.sort()
+    assert expected_files == directory_content
+
+    # Verify that the payloads are both properly restored by checking for the MZ header
+    with open(recovery_dst.joinpath(payload_filename), "rb") as payload_file:
+        header = payload_file.read(2)
+        assert header == b"MZ"
+
+    # Verify that the security descriptors are valid security descriptors
+    with open(recovery_dst.joinpath(security_descriptor_filename), "rb") as descriptor_file:
+        descriptor_buf = descriptor_file.read()
+        descriptor = SecurityDescriptor(BytesIO(descriptor_buf))
+
+        assert isinstance(descriptor.dacl, ACL)
+        assert isinstance(descriptor.sacl, ACL)
+
+        assert descriptor.owner == expected_owner
+        assert descriptor.group == expected_group
+
+    # Verify valid zone identifier for mimikatz file
+    # First parse the recovered file as an INI
+    zone_identifier = ConfigParser()
+    zone_identifier.read(recovery_dst.joinpath(zone_identifier_filename))
+
+    # Verify that sections are correct
+    assert zone_identifier.sections() == list(expected_zone_identifier.keys())
+    for section in expected_zone_identifier.keys():
+        for option, value in expected_zone_identifier[section].items():
+            # Verify that option exists for this section
+            assert option in zone_identifier.options(section)
+
+            # Verify that option has the expected value
+            assert zone_identifier.get(section, option) == value
```

Comment:
Why not do a simple string/byte comparison between the recovered zone identifier and the bytes you expect?